### ISSUE TITLE
Execution continues when passing a parameter to a function that is a terminating function call

### ISSUE THREAD

**Issue by Olipro** [ayecue/greybel-vs#145]
```Lua
f = function; exit("bye"); end function

u = function(param); print "f"; end function

u(f)
```
Despite `f` terminating the program, `u` is executed anyway.

**Comment by ayecue** [comment]
Nice find. Should be fixed with 1.10.2 - https://github.com/ayecue/greybel-vs/pull/146 merged.

**Comment by Olipro** [comment]
Nice, thanks! I noticed it while doing some tests with MiniMock :)
